P5/Source/Specs/listAnnotation.xml

```diff
@@ -0,0 +1,143 @@
+          </note>
+        </listAnnotation>
+      </standOff>
+      <!-- ... -->
+      <lg xml:id="RotAM.4.15" rhyme="ABCB">
+        <l>The self-same moment I could pray;</l>
+        <l>And from my neck so free</l>
+        <l>The albatross fell off, and sank</l>
+        <l>Like lead into the sea.</l>
+      </lg>
+    </egXML>
+  </exemplum>
+  <exemplum xml:lang="en">
+    <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#UND">
+      <listAnnotation>
+        <annotationBlock corresp="#u1">
+          <spanGrp type="pos">
+            <span from="#e14.w1" to="#e14.w1">CONJ</span>
+            <span from="#e14.w2" to="#e14.w2">RELPRO</span>
+            <span from="#e14.w3" to="#e14.w3">V</span>
+          </spanGrp>
+          <spanGrp type="prosody">
+            <span from="#T0" to="#T1">faster</span>
+          </spanGrp>
+        </annotationBlock>
+        <!-- ... annotationBlocks for #u2 through #u27 here -->
+      </listAnnotation>
+    </egXML>
+  </exemplum>
+  <exemplum xml:lang="en">
+    <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#UND">
+      <listAnnotation>
+        <annotation xml:id="bgann1" motivation="commenting" target="#match(bg-c1p1s1,'Gallia.*omnis')">
+          <respStmt>
+            <resp>creator</resp>
+            <persName>Francis Kelsey</persName>
+          </respStmt>
+          <note>‘Gaul as a whole,’ contrasted with Gaul in the narrower sense, or Celtic Gaul; Celtic Gaul also is often called Gallia.</note>
+        </annotation>
+        <annotation xml:id="bgann2" motivation="commenting" target="#match(bg-c1p1s1,'Gallia.*divisa')">
+          <respStmt>
+            <resp>creator</resp>
+            <persName>Rice Holmes</persName>
+          </respStmt>
+          <note>Gallia...divisa: Notice the order of the words. ‘Gaul, taken as a whole, is divided’.</note>
+        </annotation>
+        <annotation xml:id="bgann3" motivation="commenting" target="#match(bg-c1p1s1,'Belgae')">
+          <respStmt>
+            <resp>creator</resp>
+            <persName>Arthur Tappan Walker</persName>
+          </respStmt>
+          <note>Belgae -arum m., the Belgae or Belgians</note>
+        </annotation>
+        <annotation xml:id="bgann4" motivation="commenting" target="#match(bg-c1p1s1,'Aquitani')">
+          <respStmt>
+            <resp>creator</resp>
+            <persName>Arthur Tappan Walker</persName>
+          </respStmt>
+          <note>Aquitani, -orum m.: the Aquitani, inhabiting southwestern Gaul</note>
+        </annotation>
+        <annotation xml:id="bgann5" motivation="commenting" target="#match(bg-c1p1s1,'Celtae')">
+          <respStmt>
+            <resp>creator</resp>
+            <persName>Arthur Tappan Walker</persName>
+          </respStmt>
+          <note>Celtae, -arum m: the Celtae or Celts</note>
+        </annotation>
+        <annotation xml:id="bgann6" motivation="commenting" target="#match(bg-c1p1s1,'Gallos.*dividit')">
```

Comment:
I am not quite sure what to make of this one. Is it meant to match? If not, it should have a comment indicating such. If so, then I see two problems:

1. The string I think you are looking for is in `bg-c1p1s2`, not `bg-c1p1s1`.
1. The `.*` will match any sequence of characters _other than_ U+000A or U+000D. But (at least, using the XDM) there is a U+000A in between “Gallos” and “dividit”.